cli/progress/multiprogress.go

```diff
@@ -504,7 +564,15 @@ func (m *modelMultiProgress) View() string {
 			stats += fmt.Sprintf(" %d/%d errored", erroredAssets, len(m.Progress)-1)
 		}
 
-		pad := strings.Repeat(" ", m.maxNameWidth-len(stats))
+		if notApplicableAssets > 0 {
+			stats += fmt.Sprintf(" %d/%d n/a", notApplicableAssets, len(m.Progress)-1)
```

Comment:
nit: Here, we have "n/a" in lowercase, for the individual progress bar, it's "N/A". I think we should decide on lower or upper case, but not use both.

Reply:
I agree, lets go with lower case n/a